StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSheet/CustomerSavedPaymentMethodsCollectionViewController.swift

```diff
@@ -386,7 +386,9 @@ extension CustomerSavedPaymentMethodsCollectionViewController: UICollectionViewD
 
         cell.setViewModel(viewModel.toSavedPaymentOptionsViewControllerSelection(),
                           cbcEligible: cbcEligible,
-                          allowsPaymentMethodRemoval: configuration.paymentMethodRemove)
+                          allowsPaymentMethodRemoval: configuration.paymentMethodRemove,
+                          allowsSetAsDefaultPM: false,
+                          showDefaultPMBadge: false)
```

Comment:
I'm guessing customerSheet changes will come at a later time?

Reply:
Bella said that the CustomerSheet should not ever display a default badge because it's implied that the selected payment method in CustomerSheet is your default.